saleor/dashboard/order/forms.py

```diff
@@ -252,18 +252,25 @@ class OrderNoteForm(forms.Form):
 
 
 class ManagePaymentForm(forms.Form):
+
     amount = forms.DecimalField(
         label=pgettext_lazy(
             'Payment management form (capture, refund, void)', 'Amount'),
         max_digits=settings.DEFAULT_MAX_DIGITS,
         decimal_places=settings.DEFAULT_DECIMAL_PLACES)
 
+    clean_status = None
+
     def __init__(self, *args, **kwargs):
         self.payment = kwargs.pop('payment')
         super().__init__(*args, **kwargs)
 
     def clean(self):
-        if self.payment.charge_status != self.clean_status:
+        # Convert clean_status to a tuple if it is not a tuple or list yet
```

Comment:
Is it really needed? Why not just making it mandatory to put the values into an iterable? Those instructions are executed during the `clean()` step, it performs checks that we could prevent from being false.